CONTRIBUTING.md

```diff
@@ -1,14 +1,63 @@
 ## Build
 
-Sloop uses GitHub to manager reviews of pull requests
+Sloop uses GitHub to manage reviews of pull requests
+
+[Singing up for a GitHub account](https://docs.github.com/en/github/getting-started-with-github/signing-up-for-a-new-github-account) if you don't have one
+
+
 
 ## Steps to Contribute
 
-ADD
+1.[Pick an issue you are interested in working on](https://github.com/salesforce/sloop/issues).
+It’s a good idea to comment on an Issue that you want to help with and ask some questions before doing any major coding work
+
+2.Fork the repository: Look for the “Fork” button in the top right corner of [salesforce/sloop repo](https://github.com/salesforce/sloop)
+
+3.Clone the fork: Click on the green “clone or download” button,you can choose “clone with HTTPS.” Or, you can use SSH, which is compatible with 2-factor authentication, but just [some more work to set up](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh).
+Go to the folder where you want to save the repository, type `git clone that-https-url-you-copied`
+
+4: Add a remote pointing to the original repository so you can sync changes: `git remote add upstream git@github.com:salesforce/sloop.git`
+
+5.Make a git branch: If you type git status you will see the branch name that you’re on, called master. In most projects, master is a special place where the most stable, reviewed, up-to-date code is. So, you’ll need to make your own branch: 
+
+`git branch a-descriptive-name`
+
+Then switch to that branch:
+`git checkout name-for-your-branch`
```

Comment:
consider changing to "git checkout -b <branch-name>"

Reply:
That's a good catch, Thanks Kritika! already updated it

Reply:
@kritikapradhan good catch, `git checkout -b` is used to create AND switch to a new branch, since we've already create a new branch, `git checkout branch_name` is more reasonable to just switch to the branch